src/dashboardHandler.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const DashboardTrendGenerator = require('./dashboardGenerator');
+
+const AWS = require('aws-sdk');
```

Comment:
Can you add XRay support?

Reply:
Can be subsequent PR.

May need mocking or something in the test client to avoid XRay explosions